test/components/structures/MessagePanel-test.js

```diff
@@ -435,6 +436,8 @@ describe('MessagePanel', function() {
 
     it('should collapse creation events', function() {
         const events = mkCreationEvents();
+        const createEventMap = new Map([["", events[0]]]);
+        room.currentState.events = new Map([['m.room.create', createEventMap]]);
```

Comment:
could we get a helper for this pattern, maybe with a signature like `upsertRoomStateEvents(room.currentState, events);`